modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserGroupResourceTest.java

```diff
@@ -218,6 +221,39 @@ protected UserGroup testDeleteUserGroupUsers_addUserGroup()
 		return _postUserGroup();
 	}
 
+	@Override
+	protected UserGroup testGetAssetLibrariesUserGroupsPage_addUserGroup(
+			Long assetLibraryId, UserGroup userGroup)
+		throws Exception {
+
+		UserGroup userGroup1 = _postUserGroup(userGroup);
+
+		DepotEntry depotEntry = _depotEntryLocalService.getDepotEntry(
+			assetLibraryId);
+
+		_userGroupLocalService.addGroupUserGroups(
+			depotEntry.getGroupId(), new long[] {userGroup1.getId()});
+
+		return userGroup1;
+	}
+
+	@Override
+	protected Long testGetAssetLibrariesUserGroupsPage_getAssetLibraryId()
+		throws Exception {
+
+		DepotEntry depotEntry = _addAssetLibrary();
+
+		return depotEntry.getDepotEntryId();
+	}
+
+	@Override
+	protected Long
+			testGetAssetLibrariesUserGroupsPage_getIrrelevantAssetLibraryId()
+		throws Exception {
+
+		return null;
```

Comment:
Do we need this method to return `null`? Maybe you want to implement it.

Reply:
I do cant how this is useful, test is valid without it and there are a ton of cases where this kind of methods return a null

Reply:
However its use case seems to be _irrelevant_  :smile:

Reply:
yes, I've seen this but what is the point of using it?

I've had check the code and it seems that the assertions, for our use case do not provide any aditional test coverage, just testing the same as but with only 1 user group

Reply:
Good, the only thing is that why are you overriding the parent one wich already returns null.

Reply:
I've updated it, the getAssetLibrariesUserGroupsPage change produced some side effects in the tests